README.md

```diff
@@ -85,7 +85,7 @@ wash station's clean power and car's clean mark * car wash station
 rating / car wash station 
 distance to the center of the city, returns number rounded 
 to 1 decimal;
-3. `wash_single_car` - method, that washes a single car, so it should 
+3. `wash_single_car`s - method, that washes a single car, so it should 
```

Comment:
Remove `s` from this file, you are not supposed to commit changes to the readme.